core/config/docs/core.toml

```diff
@@ -597,3 +597,8 @@ SamplingRatio = 1.0 # Example
 [Tracing.Attributes]
 # env is an example user specified key-value pair
 env = "test" # Example
+
+[TxmAsService]
+# Enabled turns Transaction Manager as a service feature on or off. When enabled exposes endpoint to submit arbitrary EVM transaction
```

Comment:
nit: Also add that when enabled, regular Core Node services won't be run. Instead only TXM dependencies will be run.

Reply:
Right now enabled does not affect which services are running. Should I introduce these changes as part of this PR?